### PR TITLE
Network Monitor utility

### PR DESCRIPTION
**Pull request by mrpalide**
**Changes**:
- add common used values of Network Monitor
- change name of two variables in skyenv

### CODE REVIEW

File: pkg/networkmonitor/values.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package networkmonitor
+
+// WhitelistPKs store whitelisted keys of network monitor
+type WhitelistPKs map[string]struct{}
```
Comment (ersonp) [inline pkg/networkmonitor/values.go:4]: We can also move this logic `api.WhitelistPKs[v] = struct{}{}` as a `Set()` func along with this as `_, ok := WhitelistPKs[nmPkString]` `Get()`

File: pkg/skyenv/values.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ const (
 	DefaultUptimeTrackerAddr   = "http://ut.skywire.skycoin.com"
 	DefaultAddressResolverAddr = "http://ar.skywire.skycoin.com"
 	DefaultSetupPK             = "0324579f003e6b4048bae2def4365e634d8e0e3054a20fc7af49daf2a179658557"
-	DefaultNetworkMonitorPK    = ""
+	DefaultNetworkMonitorPKs   = ""
```
Comment (ersonp) [inline pkg/skyenv/values.go:34]: Since it contains a single PK `DefaultNetworkMonitorPK` is correct.

File: pkg/skyenv/values.go [hunk 3]
```diff
@@ -43,7 +43,7 @@ const (
 	TestUptimeTrackerAddr   = "http://ut.skywire.dev"
 	TestAddressResolverAddr = "http://ar.skywire.dev"
 	TestSetupPK             = "026c2a3e92d6253c5abd71a42628db6fca9dd9aa037ab6f4e3a31108558dfd87cf"
-	TestNetworkMonitorPK    = "0218905f5d9079bab0b62985a05bd162623b193e948e17e7b719133f2c60b92093"
+	TestNetworkMonitorPKs   = "0218905f5d9079bab0b62985a05bd162623b193e948e17e7b719133f2c60b92093"
```
Comment (ersonp) [inline pkg/skyenv/values.go:46]: Same as above

**Review by ersonp — APPROVED** [review]

File: pkg/skyenv/values.go (hunk 2 above)
Comment (mrpalide) [reply pkg/skyenv/values.go:34]: It could be multiple PKs later.

File: pkg/skyenv/values.go (hunk 3 above)
Comment (mrpalide) [reply pkg/skyenv/values.go:46]: Same Default

File: pkg/networkmonitor/values.go (hunk 1 above)
Comment (mrpalide) [reply pkg/networkmonitor/values.go]: Done

**Pull request merged**